.pre-commit-config.yaml

```diff
@@ -12,6 +12,14 @@ exclude: >
         ^src/sentry/data/
     )
 repos:
+-   repo: local
+    hooks:
+    - id: flake8
+      name: flake8
+      entry: flake8
+      language: python
+      types: [python]
+      log_file: '.artifacts/flake8.pycodestyle.log'
```

Comment:
note here that you can `additional_dependencies: [sentry-flake8==...]` and then you don't need to depend on it being globally installed